config/routes.rb

```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
   devise_for :users
-  resources :items
+  resources :items do
+    member do
```

Comment:
is it possible to have routes like /items/*number*/label/*width*x*height* to be more flexible?